src/components/ModiteList/styles.module.css

```diff
@@ -171,3 +171,32 @@
   top: 20px;
   left: 0px;
 }
+
+.detailsView > .moditeListWrap > div > div[class="List"] {
+  opacity: 0;
+}
+
+.detailsView ~ div[class*="MapCt"] div[class*="MapEl"] {
```

Comment:
what does this `div[class*="MapCt"]` dirty magic do?

Reply:
"Fun" way to modify the map position of a peer / child element when the ModiteList switches to a detail view for Modite or Project